### PR TITLE
Add Postman collection and environment

### PR DESCRIPTION
**Pull request by vboctor**
Fixes #25998

### CODE REVIEW

**Comment by vboctor** [comment]
@dregad do you have feedback on this one?

**Comment by dregad** [comment]
Not yet, sorry. I'm not familiar with Postman, and have not had time to install the client it and figure out how to use the files in this PR. 

I think it would be useful to have a short section in the Dev Guide detailing
- where to get & install the software
- how to open/use the included .json files
- the process to update the files and publish the documentation

**Comment by kl3sk** [comment]
I may help about that. 

I currently use postman and mantis REST API

**Comment by vboctor** [comment]
@kl3sk thanks for offering to help.  Are you suggesting to submit a PR?  If so, that would be great.

**Comment by kl3sk** [comment]
What kind of PR, this subject talks about Postman Collection and Env ?

**Comment by vboctor** [comment]
@dregad I have updated the PR with the latest Postman collection and updates to the developer manual.

**Comment by vboctor** [comment]
@dregad will merge this PR tomorrow

File: docbook/Developers_Guide/en-US/Api_Rest.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+		<literal>https://server.com/mantis/api/rest/</literal>.
+	</para>
+
+	<section id="restapi.postman">
+		<title>Postman</title>
+
+		<para>
+			Postman is a tool to enable REST API development and collaboration.
+			It is the tool that is used by MantisBT team to share the API definitions
```
[the remark this answers is not in the document]
Comment (vboctor) [reply docbook/Developers_Guide/en-US/Api_Rest.xml]: Applied the tweak to the text.

**Comment by vboctor** [comment]
@dregad I'll follow up on the open source plan and publishing issue. I have opened issues in our tracker for these follow ups.

**Comment by dregad** [comment]
Thanks !

**Comment by dregad** [comment]
> I have opened issues in our tracker for these follow ups.

For the record 
- team access https://www.mantisbt.org/bugs/view.php?id=26983
- discoverability https://www.mantisbt.org/bugs/view.php?id=26982

**Pull request merged**